Documentation/contributing/docs/docstest.rst

```diff
@@ -12,11 +12,48 @@ Documentation Testing
 First, start a local document server that automatically refreshes when you save files for
 real-time preview. It relies on the ``cilium/docs-builder`` Docker container.
 
+Set up your development environment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. Note::
+    To run Cilium's documentation locally, you need to install `docker engine <https://docs.docker.com/engine/install/>`_ and also the ``make`` package (Installed by default on Ubuntu).
+    To verify that ``make`` and ``docker`` is installed, run the command ``make --version`` and ``docker --version`` in your terminal.
+
+.. code-block:: shell-session
+
+    $ docker --version
+    Docker version 20.10.22, build 3a2c30b
+    $ make --version
+    GNU Make 4.2.1
+
+For Windows
+~~~~~~~~~~~
+
+.. Note::
+    The preferred method is to upgrade to Windows 10 version 1903 Build 18362 or higher, you can upgrade to Windows Subsystem for Linux ``WSL2`` and run ``make`` in Linux.
+
+* Verify you have access to the ``make`` command in your terminal.
+* Download and install docker desktop.
+* Set up docker to use `WSL2 <https://docs.docker.com/desktop/windows/wsl/>`_ as backend.
+* Start docker desktop.
```

Comment:
If you use `#.` for these bulletpoints, then Sphinx will turn it into an enumerated list, that way it is clear that these are a set of linear steps:
```suggestion
#. Verify you have access to the ``make`` command in your terminal.
#. Download and install docker desktop.
#. Set up docker to use `WSL2 <https://docs.docker.com/desktop/windows/wsl/>`_ as backend.
#. Start docker desktop.
```